.github/workflows/nightly-browser-view.yml

```diff
@@ -0,0 +1,229 @@
+      MACOS_API_KEY: ${{ secrets.MM_DESKTOP_MAC_APP_STORE_MACOS_API_KEY }}
+      MACOS_API_ISSUER_ID: ${{ secrets.MM_DESKTOP_MAC_APP_STORE_MACOS_API_ISSUER_ID }}
+      CSC_FOR_PULL_REQUEST: true
+      CSC_KEY_PASSWORD: ${{ secrets.MM_DESKTOP_MAC_APP_STORE_CSC_KEY_PASSWORD}}
+      CSC_LINK: ${{ secrets.MM_DESKTOP_MAC_APP_STORE_CSC_LINK }}
+    needs:
+      - begin-notification
+    steps:
+      - name: nightly/checkout-repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: nightly/setup-node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: "package.json"
+          cache: "npm"
+          cache-dependency-path: package-lock.json
+      - name: nightly/create-build-folder
+        run: mkdir -p ./build
+      - name: nightly/install-dependencies
+        run: |
+          brew install yq
+          npm ci
+      - name: nightly/copy-provisioning-profile
+        run: echo $MAS_PROFILE | base64 -D > ./mas.provisionprofile
+      - name: nightly/patch-version-number-for-MAS
+        run: ./scripts/patch_mas_version.sh
+      - name: nightly/test
+        uses: ./.github/actions/test
+      - name: nightly/package
+        run: npm run package:mas
+      - name: nightly/publish
+        run: fastlane publish_test path:"$(find . -name \*.pkg -print -quit)"
+
+  build-mac-installer:
+    runs-on: macos-12
+    needs:
+      - mac-app-store-preflight
+    steps:
+      - name: nightly/checkout-repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: nightly/setup-node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: "package.json"
+          cache: "npm"
+          cache-dependency-path: package-lock.json
+      - name: nightly/create-build-folder
+        run: mkdir -p ./build
+      - name: nightly/install-dependencies
+        run: |
+          brew install yq rename
+          npm ci
+      - name: nightly/test
+        uses: ./.github/actions/test
+      - name: nightly/patch-version
+        uses: ./.github/actions/patch-nightly-version
+      - name: nightly/build
+        env:
+          APPLEID: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_APPLE_ID }}
+          APPLEIDPASS: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_APPLE_ID_PASS }}
+          CSC_FOR_PULL_REQUEST: true
+          CSC_KEY_PASSWORD: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_CSC_KEY_PASSWORD }}
+          CSC_LINK: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_CSC_LINK }}
+        run: |
+          mkdir -p ./build/macos-release
+          npm run package:mac-with-universal
+          bash -x ./scripts/patch_updater_yml.sh
+          bash -x ./scripts/cp_artifacts.sh release ./build/macos-release
+      - name: nightly/rename-arm64-to-m1
+        run: rename 's/arm64/m1/' ./build/macos-release/$(jq -r .version package.json)/*
```

Comment:
Maybe decouple the .version extract with a variable, and use the Variable to inject this to the command. 
Would something like the following work: ? 
```
env:
  APP_VERSION: $(jq -r .version package.json))
```